### ISSUE TITLE
Drag and Drop python files in spyder

### ISSUE THREAD

**Issue by markushartmann** [spyder-ide/spyder#6916]
## Problem Description

Drag and Drop files from finder to spyder not possible

### What steps will reproduce the problem?

1. take a python file in finder window
2. switch window to spyder
3. drop it into editor area

### What is the expected output? What do you see instead?

Expected behaviour is to see the file in the editor. 
I see now new file in the editor but in the Internal console a error message.

### Paste Traceback/Error Below (if applicable)

```python-traceback

Traceback (most recent call last):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/spyderlib/widgets/editor.py", line 1891, in dragEnterEvent
    text = [encoding.is_text_file(url) for url in all_urls]
TypeError: 'NoneType' object is not iterable

```

## Package Versions

* Spyder: 2.3.8 installation from macports
* Python:  2.7.14 64bits
* Qt: 4.8.7
* PyQt: PyQt4 (API v2) 4.12.1 on Darwin
* Operating System: MacOS 10.12.6

### Dependencies

```
IPython >=1.0      :  5.4.0 (OK)
jedi >=0.8.1;<0.9.0:  0.11.1 (NOK)
matplotlib >=1.0   :  2.1.1 (OK)
pandas >=0.13.1    :  0.22.0 (OK)
pep8 >=0.6         :  1.7.0 (OK)
psutil >=0.3       :  5.4.3 (OK)
pyflakes >=0.5.0   :  1.6.0 (OK)
pygments >=1.6     :  2.2.0 (OK)
pylint >=0.25      :  1.8.3 (OK)
qtconsole >=4.0    :  4.3.1 (OK)
rope >=0.9.2       :  0.10.6 (OK)
sphinx >=0.6.6     :  1.7.2 (OK)
sympy >=0.7.3      :  1.1.1 (OK)
zmq >=2.1.11       :  17.0.0 (OK)

```

**Comment by markushartmann** [comment]
The full output is:

`
/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/nbformat/current.py:19: UserWarning: nbformat.current is deprecated.

- use nbformat for read/write/validate public API
- use nbformat.vX directly to composing notebooks of a particular version

  """)
Traceback (most recent call last):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/spyderlib/widgets/editor.py", line 1891, in dragEnterEvent
    text = [encoding.is_text_file(url) for url in all_urls]
TypeError: 'NoneType' object is not iterable`
'

**Comment by ccordoba12** [comment]
> Spyder: 2.3.8 installation from macports

We don't support this version anymore. It's almost three years old.

Please update to our most recent version (3.2.8) and try again.

**Comment by CAM-Gerlach** [comment]
This is a duplicate of #5218 ; fixed in Spyder 3.2.5. You can update with ``conda update spyder`` from the Anaconda Prompt.
